package.json

```diff
@@ -58,11 +58,13 @@
     "@lightspeed/config-prettier": "^0.1.2",
     "@lightspeed/config-typescript": "^0.2.0",
     "@lightspeed/eslint-config": "^1.1.1",
-    "@percy/storybook": "^3.2.0",
-    "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-docs": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
-    "@storybook/react": "^5.3.19",
+    "@percy/storybook": "^3.3.0",
+    "@storybook/addon-actions": "6.0.0-rc.10",
+    "@storybook/addon-docs": "6.0.0-rc.10",
+    "@storybook/addons": "6.0.0-rc.10",
+    "@storybook/api": "^5.3.19",
+    "@storybook/components": "^5.3.19",
```

Comment:
Issues with `6.0.0-rc.10` on those two?

Reply:
good catch, should also be on the latest RC